testing/toolchain/toolchain.bzl

```diff
@@ -14,5 +15,8 @@ scala_testing_toolchain = rule(
             ],
             providers = [_DepsInfo],
         ),
+        "jacocorunner": attr.label(
```

Comment:
If we have `jacocorunner` on `scala_toolchain`, then it's not needed in `testing_toolchain`

Reply:
Sure. Will fix.